From 9d76ba39ee466d6c1af5b26bcdc597b67d6fbf24 Mon Sep 17 00:00:00 2001
From: Neebe3289 <neebexd@gmail.com>
Date: Thu, 26 Mar 2026 22:47:21 +0700
Subject: [PATCH] kernel/apk_sign: Restore RKSU manager

* rissu said will migrate his base on top of backslashxx/xxksu

Change-Id: I483b415910b3d183456958a88a67040c663a6b84
Signed-off-by: Neebe3289 <neebexd@gmail.com>
---
 kernel/apk_sign.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/apk_sign.c b/kernel/apk_sign.c
index 8686b543..d72cf8bf 100644
--- a/kernel/apk_sign.c
+++ b/kernel/apk_sign.c
@@ -382,6 +382,6 @@ bool is_manager_apk(char *path)
 	|| check_v2_signature(path, EXPECTED_SIZE, EXPECTED_HASH)  // kernelsu official
 	|| check_v2_signature(path, 0x375, "484fcba6e6c43b1fb09700633bf2fb4758f13cb0b2f4457b80d075084b26c588")  // KOWX712/KernelSU
 	|| check_v2_signature(path, 0x3e6, "79e590113c4c4c0c222978e413a5faa801666957b1212a328e46c00c69821bf7")  // rifsxd/KernelSU-Next
-//	|| check_v2_signature(path, 0x396, "f415f4ed9435427e1fdf7f1fccd4dbc07b3d6b8751e4dbcec6f19671f427870b")  // rsuntk/KernelSU, temp remove, reason: old handle_sepolicy
+	|| check_v2_signature(path, 0x396, "f415f4ed9435427e1fdf7f1fccd4dbc07b3d6b8751e4dbcec6f19671f427870b")  // rsuntk/KernelSU
 	);
 }
-- 
2.52.0

